doc/services/device_mgmt/ec_host_cmd.rst

```diff
@@ -0,0 +1,64 @@
+.. _ec_host_cmd_backend_api:
+
+EC Host Command
+###############
+
+Overview
+********
+EC Host Command is a subsystem used to communicate with a host via the host command protocol.
+Only version 3 of the protocol is supported.
```

Comment:
"The host command protocol defines the interface for a host, or application processor, to communicate with a target embedded controller (EC).  The EC Host command subsystem implements the target side of the protocol, generating responses to commands sent by the host.   The host command protocol interface supports multiple versions, but this subsystem implementation only support protocol version 3."

Reply:
Done.